pyrometheus/__init__.py

```diff
@@ -868,42 +523,43 @@ def get_temperature(self, enthalpy_or_energy, t_guess, y, do_energy=False):
             j = -pv_fun(t_i, y)
             dt = -f / j
             t_i += dt
-            if self.npctx.linalg.norm(dt, np.inf) < tol:
+            if _pyro_norm(self.usr_np, dt, np.inf) < tol:
                 break
 
         return t_i
 
     def get_falloff_rates(self, T, C, k_fwd):
-        k_high = make_obj_array([
+        ones = (1.0 + T) - T
```

Comment:
I've seen this is a couple of places now, and I'm just curious as to what it does? Is it a way of creating an array of 1's?

Reply:
It's 1's, but importantly respecting the size, shape, and nature (let's say) of *T*, whatever *T* may be.